### ISSUE TITLE
Some Options are overrided in Sortable.create

### ISSUE THREAD

**Issue by archgeko** [nextapps-be/livewire-sortablejs#22]
I noticed that some options are overrided after you spread the "options" object
```javascript
el.livewire_sortable = window.Sortable.create(el, {
        ...options,
        draggable: '[wire\\:sortable-group\\.item]',
        handle: el.querySelector('[wire\\:sortable-group\\.handle]') ? '[wire\\:sortable-group\\.handle]' : null,
        sort: true, 
        dataIdAttr: 'wire:sortable-group.item',
        group: {
            name: el.closest('[wire\\:sortable-group]').getAttribute('wire:sortable-group'),
            pull: true,
            put: true,
        },
....
}
``` 

Options like "sort" and "group.pull", "group.put" are set After the spread operator on options and overrides the ones you set in your blade

GC

**Comment by gdebrauwer** [comment]
Fixed #23

**Comment by boothy060590** [comment]
The "sort" option is still overidden. This prevents us from setting up grouped lists that are unsortable but items can cloned from one another.

**Comment by gdebrauwer** [comment]
@boothy060590 This has been fixed in v0.3.2 (and v0.2.2)
